app/views/layouts/application.html.erb

```diff
@@ -69,6 +69,17 @@
 		</div><!--/.container-fluid -->
 	</div>
 
+	<% if notice %>
+	<br/>
```

Comment:
These line breaks were unnecessary. I have just seen the preview.

I have removed them before merging.